README.md

```diff
@@ -24,3 +25,20 @@ answer, including your reasoning, to this markdown file.
 ## Bonus
 
 Implement and analyze breadth-first search.
+
+# Response
+
+This exercise was a tough one for me, mostly because of an issue I had with using recursion within for loops. The loop variable would persist in scope through recursive layers, causing all kinds of odd behavior. Thankfully, Cade figured out a solution to that problem for me. I did rebuild my solution to this 4 separate times before I got it working the way I wanted. Now, I feel good about it.
+
+## Runtime Analysis
+
+For my implementation in the worst case it will iterate over each node, and each edge connected to each node. It won't iterate over a node more than once, because it stores which nodes it has visited. Therefore, it should have a runtime complexity of $|V|*|E|$ in the worst case.
```

Comment:
Hmm, so it would traverse each edge for each node?

Reply:
I guess it wouldn't actually traverse each edge for each node, but each edge which it could have to another node, which would be equal to the number of nodes. So that should instead be the number of nodes. This would then be a runtime of $|V^2|$ in complexity, right?